### PR TITLE
Add Dockerfiles for SLES15

### PR DESCRIPTION
**Pull request by mscastanho**
Added Dockerfiles to all AT versions that support SLES15 (AT 13.0 and up).

Signed-off-by: Matheus Castanho <msc@linux.ibm.com>

### CODE REVIEW

File: dockerfile-suse.template [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+WORKDIR /root
+
+RUN zypper refresh \
+    && zypper install -y __BASE_PACKAGES__ \
+    && zypper clean
+
+RUN rpm --import ${REPO}/${DISTRO_NAME}/SLES_${DISTRO_NICK}/gpg-pubkey-6976a827-5164221b \
+    && zypper addrepo ${REPO}/suse/SLES_15/ AT \
+    && zypper install -y __AT_PACKAGES__
```
Comment (ThinkOpenly) [inline dockerfile-suse.template]: Why do you perform `zypper clean` before the later `zypper install`, and not after?

Comment (mscastanho) [reply dockerfile-suse.template]: @ThinkOpenly Good point. I'll move it to after the second install command.

File: dockerfile-suse.template [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+RUN rpm --import ${REPO}/${DISTRO_NAME}/SLES_${DISTRO_NICK}/gpg-pubkey-6976a827-5164221b \
+    && zypper addrepo ${REPO}/suse/SLES_15/ AT \
+    && zypper install -y __AT_PACKAGES__ \
+    && zypper clean
+
+RUN ln -sf /opt/at${AT_VERSION}${MINOR}${EXTRA}/lib64/ld64.so.2 /lib64/ld64.so.2
+RUN ln -sf /opt/at${AT_VERSION}${MINOR}${EXTRA}/sbin/ldconfig /sbin/ldconfig
+
+ENV PATH /opt/at${AT_VERSION}${MINOR}${EXTRA}/bin:$PATH
```
Comment (ThinkOpenly) [inline dockerfile-suse.template]: Just a thought, for your consideration...

We have environment module files included with AT, although they may not be well known.  They can be used to set up an environment to run AT, and today they set not only PATH, as you have done here, but also MANPATH and INFOPATH, which are convenient.  If new settings arise in the future, we could add those, too.  The disadvantage is that another package must be installed before invoking the command shell:  environment-modules.

If the environment-modules package is installed, then appending `module load at${AT_VERSION}${MINOR}${EXTRA}` to `.bash_profile` I think would be sufficient.

Comment (mscastanho) [reply dockerfile-suse.template]: @ThinkOpenly Good point. Fixed. I also fixed some symlinks that were not pointing to the correct files.

**Review by ThinkOpenly — APPROVED** [review]

**Pull request merged**